### ISSUE TITLE
When exporting to docker unpublished hab-pkg-dockerize is being used

### ISSUE THREAD

**Issue by bodymindarts** [habitat-sh/habitat#2802]
```
$ hab -V
hab 0.26.1/20170715032253
```

In studio:
```
 hab pkg export docker core/kafka
∵ Missing package for core/hab-pkg-dockerize
» Installing core/hab-pkg-dockerize
↓ Downloading core/hab-pkg-dockerize/0.27.0-dev/20170722192835
    4.12 KB / 4.12 KB / [=====================================================================================================================================] 100.00 % 92.02 KB/s
☛ Verifying core/hab-pkg-dockerize/0.27.0-dev/20170722192835
→ Using core/acl/2.2.52/20170513213108
```

As you can see `core/hab-pkg-dockerize/0.27.0-dev` is being pulled. I would expect it to be `core/hab-pkg-dockerize/0.26.1` to match the hab cli version or at least the latest published one.

**Comment by eeyun** [comment]
It's possible that this is related to the issue https://github.com/habitat-sh/habitat/issues/2810

**Comment by reset** [comment]
Fixed in 0.28.0
